ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml

```diff
@@ -38,9 +34,11 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    test:
-    - chain: redhat-chaos-krkn-hub-tests
-    - chain: redhat-chaos-hog-scenarios
+    workflow: redhat-chaos-krkn-hub-tests
```

Comment:
you can only specify 1 workflow per test section

Reply:
and a workflow can't be under tests (line 39) only ref or chains

Reply:
I saw workflow being used under test in the [periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests-arm] file...so I thought I'll try it

Reply:
yeh but you list 2 of them. can only have 1

Reply:
Yes yes....I ran the make ci-operator-config job and it removed one of the workflow